src/main/kotlin/racingcar/domain/Cars.kt

```diff
@@ -2,9 +2,12 @@ package racingcar.domain
 
 class Cars(carNames: List<String>) : List<Car> by (carNames.map { name -> Car(name) }) {
```

Comment:
아래와 같은 주 생성자(primary constructor)를 만든 후에 자기 생성자 호출하는 `this()`를 사용하면 어떨까요?

```suggestion
class Cars(private val cars: List<Car>) : List<Car> by cars {
```